library/Fission/Platform/User.hs

```diff
@@ -24,10 +24,10 @@ data UserConfig = UserConfig
              , Generic
              )
 
-makeLenses ''UserConfig
-$(deriveJSON lens_SCREAMING_SNAKE_CASE ''UserConfig)
+makeLenses ''Config
+$(deriveJSON lens_SCREAMING_SNAKE_CASE ''Config)
```

Comment:
FYI, I'm going to trash any `deriveJSON` that uses options. I really dislike the entire `Fission.Internal.JSON` module. Not for this PR, but happening eventually.